prometheus-to-sd/config/gce_config.go

```diff
@@ -35,39 +50,62 @@ type GceConfig struct {
 
 // GetGceConfig builds GceConfig based on the provided prefix and metadata server available on GCE.
 func GetGceConfig(zone string, monitoredResourceTypes string) (*GceConfig, error) {
+	if *ProjectFlag != "" {
+		glog.Infof("Using metadata all from flags")
```

Comment:
I think it's worth to log a warning if any of the values: cluster name, cluster location, node name is empty - usually it won't be intended.

Reply:
sure, done.